server/utils/db/db_utils.py

```
@@ -1501,6 +1503,21 @@ def initialize_tables():
                 conn.rollback()
 
+            # Add google_chat_message_name column to incidents table for Google Chat message updates
+            try:
+                cursor.execute(
+                    """
+                    ALTER TABLE incidents
+                    ADD COLUMN IF NOT EXISTS google_chat_message_name VARCHAR(255);
+                    """
+                )
+                conn.commit()
+            except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)